ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -64,7 +64,6 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      ENABLE_OTA_TEST: OCP-23799
```

Comment:
Is the removal here expected ?

Reply:
Yeah, I requested that [here](https://github.com/openshift/release/pull/54114#discussion_r1671779422).

Reply:
Ack